builder/proxmox/builder.go

```diff
@@ -71,7 +71,7 @@ func (b *Builder) Run(ctx context.Context, ui packer.Ui, hook packer.Hook) (pack
 		},
 		&communicator.StepConnect{
 			Config:    &b.config.Comm,
-			Host:      getVMIP,
+			Host:      commHost(b.config.Comm.SSHHost),
```

Comment:
Don't we need to take the WinRM communicator into consideration here as well?

Reply:
Good catch! Fortunately `communicator.Config.Host()` takes care of it by default!

Reply:
:+1: